src/content/changes/highlights/2023-02-16.md

```diff
@@ -9,7 +9,12 @@ owner:
 
 
 ## User Interfaces
-
+- [coredns-app](https://github.com/giantswarm/coredns-app) [v1.14.2](https://github.com/giantswarm/coredns-app/releases/tag/v1.14.2) changes the Pod Disruption Budget from `maxUnavailable: 1` to `maxUnavailable: 25%` for better scaling. Drop static `replicas`, managed by HPA from the Deployment and add changes to the Config Map by adding 5 seconds to the Health Check
+- [k8s-dns-node-cache-app](https://github.com/giantswarm/k8s-dns-node-cache-app) [v2.0.0](https://github.com/giantswarm/k8s-dns-node-cache-app/releases/tag/v2.0.0) introduces support for Cilium CNI in EBPf mode and should be **only** used with AWS v19 release
+- [kong-app](https://github.com/giantswarm/kong-app) [v3.0.0](https://github.com/giantswarm/kong-app/releases/tag/v3.0.0) This release upgrades kong to release [3.1.1](https://github.com/Kong/kong/blob/3.1.1/CHANGELOG.md#311) which contains breaking changes. Please consider reading the upstream [Breaking changes](https://github.com/Kong/kong/blob/3.1.1/CHANGELOG.md#breaking-changes) documentation before upgrading. Users of the kong enterprise should read the [Kong Gateway changelogs of releases 3.0.0.0 onwards](https://docs.konghq.com/gateway/changelog/#3000) up to 3.1.1.3. Align with upstream chart version, Update kong/kong to [3.1.1](https://github.com/Kong/kong/blob/3.1.1/CHANGELOG.md#311). Please read [Breaking changes](https://github.com/Kong/kong/blob/3.1.1/CHANGELOG.md#breaking-changes) notes. Update kong ingress controller to [2.8.1](https://github.com/Kong/kong/blob/2.8.1/CHANGELOG.md#281) and Update custom resource definitions to those used by kong ingress controller [2.8.1]. **Changes:** Increase default requests/limits to 1 cpu and 2G memory and Remove startupProbe from kong ingress controller. [More information](https://github.com/Kong/charts/pull/527#issuecomment-1014782921). **Breaking Changes:** The `minReadySeconds` value previous found at `deployment.kong.minReadySeconds` is now configured at path `deployment.minReadySeconds`
```

Comment:
```suggestion
- [kong-app](https://github.com/giantswarm/kong-app) [v3.0.0](https://github.com/giantswarm/kong-app/releases/tag/v3.0.0) This release upgrades Kong to release [3.1.1](https://github.com/Kong/kong/blob/3.1.1/CHANGELOG.md#311) which contains breaking changes. Please consider reading the upstream [Breaking changes](https://github.com/Kong/kong/blob/3.1.1/CHANGELOG.md#breaking-changes) documentation before upgrading. Users of the Kong enterprise should read the [Kong Gateway changelogs of releases 3.0.0.0 onwards](https://docs.konghq.com/gateway/changelog/#3000) up to 3.1.1.3. Align with the upstream chart version, Update kong/kong to [3.1.1](https://github.com/Kong/kong/blob/3.1.1/CHANGELOG.md#311). Please read [Breaking changes](https://github.com/Kong/kong/blob/3.1.1/CHANGELOG.md#breaking-changes) notes. Update the Kong ingress controller to [2.8.1](https://github.com/Kong/kong/blob/2.8.1/CHANGELOG.md#281) and Update custom resource definitions to those used by the Kong ingress controller [2.8.1]. **Changes:** Increase default requests/limits to 1 CPU and 2G memory and removes `startupProbe` from the Kong ingress controller. [More information](https://github.com/Kong/charts/pull/527#issuecomment-1014782921). **Breaking Changes:** The `minReadySeconds` value previously found at `deployment.kong.minReadySeconds` is now configured at path `deployment.minReadySeconds`
```

Reply:
I have suggest some changes but it still takes a lot of effort for me to get the message. I see `https://github.com/Kong/kong/blob/3.1.1/CHANGELOG.md#breaking-changes` repeated twice and it mentions upgrade to `3.1.1` and later `2.8.1`. Can we split or simplify it?

Reply:
@weatherhog what's your input based on Fer comment above?

Reply:
@pipo02mix I think it makes sense to mention the breaking changes multiple times just to be sure that everyone is aware that there are breaking changes. The mentioned upgrade to `3.1.1` is regarding kong and the mentioned update to `2.8.1`is regarding kong-ingress-controller. Splitting it just would make it even more complex because the release is updating several kong components at once